src/test/java/core/basesyntax/strategy/SupplyOperationHandlerTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.FruitDto;
+import core.basesyntax.model.Fruit;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SupplyOperationHandlerTest {
+    private static OperationHandler handler;
+    private static Fruit apple;
+    private static Fruit banana;
+    private static Fruit orange;
+
+    @BeforeClass
+    public static void beforeClass() {
+        handler = new SupplyOperationHandler();
+        apple = new Fruit("apple");
+        banana = new Fruit("banana");
+        Storage.storage.put(apple, 35);
+        Storage.storage.put(banana, 8);
```

Comment:
let's do this in the `@Before` method
and after each test we should clear the storage

same in other handlerTests - fill the storage in `@Before` and clear in `@After`